repos/fdbt-site/src/pages/products/pointToPointProducts.tsx

```diff
@@ -0,0 +1,147 @@
+                            <td className="govuk-table__cell">{product.productDescription}</td>
+                            <td className="govuk-table__cell">{product.validity}</td>
+                            <td className="govuk-table__cell">{moment.utc(product.startDate).format('DD/MM/YYYY')}</td>
+                            <td className="govuk-table__cell">{moment.utc(product.endDate).format('DD/MM/YYYY')}</td>
+                            <td className="govuk-table__cell">{getTag(product.startDate, product.endDate)}</td>
+                        </tr>
+                    ))
+                ) : (
+                    <span className="govuk-body">
+                        <i>You currently have no products against this service</i>
+                    </span>
+                )}
+            </tbody>
+        </table>
+    );
+};
+
+const getTag = (startDate: string, endDate: string): JSX.Element => {
+    const today = new Date().setHours(0, 0, 0, 0);
+    const startDateAsUnixTime = moment.utc(startDate, 'DD/MM/YYYY').valueOf();
+    const endDateAsUnixTime = moment.utc(endDate, 'DD/MM/YYYY').valueOf();
+
+    if (startDateAsUnixTime <= today && endDateAsUnixTime >= today) {
+        return <strong className="govuk-tag govuk-tag--turquoise">Active</strong>;
+    } else if (startDateAsUnixTime >= today) {
+        return <strong className="govuk-tag govuk-tag--blue">Pending</strong>;
+    } else {
+        return <strong className="govuk-tag govuk-tag--red">Expired</strong>;
+    }
+};
+
+export const filterProductsNotToDisplay = (service: MyFaresService, products: MyFaresProduct[]): MyFaresProduct[] => {
+    const serviceStartDate = moment.utc(service.startDate, 'DD/MM/YYYY').valueOf();
+    const serviceEndDate = moment.utc(service.endDate, 'DD/MM/YYYY').valueOf();
+    return products.filter((product) => {
+        const productStartDate = moment.utc(product.startDate).valueOf();
+        const productEndDate = moment.utc(product.endDate).valueOf();
+        return productEndDate >= serviceStartDate && productStartDate <= serviceEndDate;
+    });
+};
+
+export const getServerSideProps = async (
+    ctx: NextPageContextWithSession,
+): Promise<{ props: PointToPointProductsProps }> => {
+    const noc = getAndValidateNoc(ctx);
+    const serviceId = ctx.query?.serviceId;
+    if (!serviceId || isArray(serviceId)) {
+        throw new Error('Unable to find line name to show products for.');
+    }
+    const service = await getBodsServiceByNocAndId(noc, serviceId);
+    const products = await getPointToPointProductsByLineId(noc, service.lineId);
+    const productsToDisplay = filterProductsNotToDisplay(service, products);
+    const formattedProducts = await Promise.all(
+        productsToDisplay.map(async (product) => {
+            const matchingJson = await getMatchingJson(product.matchingJsonLink);
+            const productDescription = `${upperFirst(matchingJson.passengerType)} - ${matchingJson.type} ${
```

Comment:
Should matchingJson.type be sentence case?

Reply:
I don't think so, I think we want it to say "Adult - return" rather than "Adult - Return" as thats more start case. Could be wrong, what do you think?

Reply:
What about flatFare ? Would have thought this should be 'flat fare' or 'Flat Fare' or something

Reply:
This page is just point to point products, the ones like flat fare and stuff will use their product name I think, on a different page.